scripts/release/utils/workspace.ts

```diff
@@ -39,21 +45,20 @@ function mapWorkspaceToPackages(workspaces) {
     )
   ).then(paths => paths.reduce((arr, val) => arr.concat(val), []));
 }
-exports.mapWorkspaceToPackages = mapWorkspaceToPackages;
 
-function mapPackagestoPkgJson(packagePaths) {
+function mapPackagestoPkgJson(packagePaths: string[]) {
   return packagePaths
     .map(path => {
       try {
-        return require(`${path}/package.json`);
+        return JSON.parse(readFileSync(`${path}/package.json`, 'utf8'));
```

Comment:
Just out of curiosity is it because ts-node can't require json files?

Reply:
`require` caches the content of the package.json, so you will read stale `package.json` if you read it first, update it, then read it again. using `readFile` will always get the latest version from disk.